docs/developing_terraform_modules.md

```diff
@@ -0,0 +1,160 @@
+## Developing Terraform Modules
+
+This document is not actually about Terraform or how to write good Terraform code or how
```

Comment:
I would reverse the order of these two statements. First what it is, then what it's not.